magpie-aws/src/main/java/io/openraven/magpie/plugins/aws/discovery/services/S3Discovery.java

```diff
@@ -222,7 +222,6 @@ private void discoverPublic(S3Client client, Bucket resource, MagpieAwsResource
       if (!(ex.statusCode() == 403 || ex.statusCode() == 404)) {
         throw ex;
       }
-      logger.warn("Failure on S3 public access discovery, BucketName: {}, Reason: {}", resource.name(), ex.getMessage());
```

Comment:
I would probably move these to debug instead.

Reply:
Done